eva-accession-import-dbsnp2/src/test/resources/application.properties

```diff
@@ -2,8 +2,10 @@ spring.batch.job.names=IMPORT_DBSNP_JSON_VARIANTS_JOB
 
 # dbSNP JSON input source
 parameters.input=src/test/resources/input-files/test-dbsnp.json.bz2
-# reference sequence accession
-parameters.assembly=GRCh38.p13
+# Genbank accession
+parameters.genbankAssembly=GCA_000001405.27
+# Reference sequence accession
```

Comment:
To make clear it refers to the service and not the common noun "reference accession":
```suggestion
# RefSeq accession
```